extensions/ext-tiled/src/main/java/org/pixel/ext/tiled/view/TiledGenericMapView.java

```diff
@@ -0,0 +1,35 @@
+package org.pixel.ext.tiled.view;
+
+import org.pixel.ext.tiled.content.TiledTileLayer;
+import org.pixel.ext.tiled.content.TiledMap;
+import org.pixel.ext.tiled.content.TiledImageLayer;
+import org.pixel.ext.tiled.content.TiledObjectGroup;
+
+/**
+ * A generic map view.
+ */
+public interface TiledGenericMapView extends TiledView<TiledMap> {
+    /**
+     * Draws a tile layer.
+     * @param layer The layer to draw.
+     */
+    default void draw(TiledTileLayer layer) {
```

Comment:
since all implementations have these functions, is there any reason for them to have a default here? (check similar ones)